### PR TITLE
Duplicate array Fixes #777

### LINKED ISSUE #777: Find Duplicate in Array
Hi, I am GSSoC participant,I would like to add this problem with Time and Space Complexity.

### PR DESCRIPTION
**Pull request by krishna-NIT**
Fixes #777 @rish-singhal @tarun26091999 @yasharth291 please review it and merge it soon

### CODE REVIEW

**Comment by krishna-NIT** [comment]
@Rohan27s  Please Review it and Let me Know if any changes

File: Array/readme.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To create an array, you need to known the length (or size) of the array in advan
  * Inverse of an array ----> [C++](Code/C++/inverse_of_an_array.cpp) | [Python](/Code/Python/inverseArray.py)
  * Job Sequencing Problem ----> [C++](/Code/C++/job_sequencing_problem.cpp)
  * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp) 
- * Finding All subset ----> [C++](/Code/C++/Finding_all_subset.cpp)
```
Comment (Rohan27s) [inline Array/readme.md:35]: why have you removed this line?

**Review by Rohan27s — CHANGES_REQUESTED** [review]
Everything else is fine, just resolve the issue in readme

**Comment by krishna-NIT** [comment]
> Everything else is fine, just resolve the issue in readme

Done Sir ! @Rohan27s

**Review by Rohan27s — APPROVED** [review]

File: Array/readme.md (hunk 1 above)
Comment (krishna-NIT) [reply Array/readme.md:34]: present at line 29

**Pull request merged**